CMakeLists.txt

```diff
@@ -494,6 +496,9 @@ if(PROJECT_PACKAGE_DEPENDENCIES)
       endif()
     elseif(${dep} STREQUAL yaml-cpp)
       get_filename_component(prefix ${YAML_CPP_INCLUDE_DIR}/.. ABSOLUTE)
+    elseif(${dep} STREQUAL Vmc)
```

Comment:
If the find_package argument is changed to `VMC`, then also change it here.

Reply:
Done